### DISCUSSION TITLE
How I can get a BaseUrl on liquid page ?

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#6994] [Q&A] [answered]
I need site url like "https://sitename.com/"
I use exemple from https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/#site but it doesn't work, I try ``{{Site.BaseUrl}}``, ``{{BaseUrl}}``, etc.

**Comment by Skrypt** [comment, marked as answer]
I just tried and it definitely works.
You are probably missing some assembly reference.

**Reply by Diman96** [reply]
What assembly reference needed for this?

**Reply by deanmarcussen** [reply]
Did you set the `BaseUrl` property in the admin? 

The general site settings can be found under _Configuration -> General -> BaseUrl_

`{{ Site.BaseUrl }}` is the correct usage

**Reply by Diman96** [reply]
Thank you, this field was empty, now it works.
